### ISSUE TITLE
Can we please add a license to this project?

### ISSUE THREAD

**Issue by BrittanyIstenes** [NateV/Expungement-Generator#24]
Apache 2.0, MIT or BSD-3 or BSD-2 are permissive licenses and will be the best fit for this project.  I can submit a PR for this

**Comment by NateV** [comment]
Yeah, that'd be great. I'd be happy to accept a PR like that. I'm interested in hearing about your interest in this project - its in the (slow) process of being replaced by https://github.com/CLSPhila/RecordLib
